.gitlab-ci.yml

```diff
@@ -21,6 +21,7 @@ variables:
   STATIC_BINARIES_DIR: bin/static
   DOGSTATSD_BINARIES_DIR: bin/dogstatsd
   AGENT_BINARIES_DIR: bin/agent
+  CLUSTER_AGENT_BINARIES_DIR: bin/datadog-cluster-agent
```

Comment:
we have `bin/agent` so it could simply be `bin/cluster-agent` ?

Reply:
I tried to be more explicit, as the notion of cluster agent could be a cluster agent from a different vendor (and it's tied to where I build the bin

Reply:
So this is used in
`$SRC_PATH/$AGENT_BINARIES_DIR/datadog-cluster-agent $S3_ARTEFACTS_URI/datadog-cluster-agent` to copy the binary. The binary is generated at this PATH:
`/src/github.com/DataDog/datadog-agent/bin/datadog-cluster-agent/datadog-cluster-agent`. But this is from the `inv cluster-agent.build` line 134.
So if I change that, I'll need to change where the binary is created.